content/client-lib-development-guide/features.textile

```diff
@@ -406,16 +407,16 @@ h3(#realtime-channel). Channel
 * @(RTL1)@ As soon as a @Channel@ becomes attached, all incoming messages and presence messages are processed and emitted where applicable. @PRESENCE@ and @SYNC@ messages are passed to the @Presence@ object ensuring it maintains a map of current members on a channel in realtime
 * @(RTL2)@ EventEmitter and states:
 ** @(RTL2a)@ Implements @EventEmitter@ and emits events for state changes @INITIALIZED@, @ATTACHING@, @ATTACHED@, @DETACHING@, @DETACHED@, @SUSPENDED@ and @FAILED@
+** @(RTL2g)@ Additionally, an @UPDATE@ event can be emitted by the @Channel@ that contains a @ChannelStateChange@ object. This is used for changes to channel conditions for which the main channel state (e.g. @ATTACHED@, per @RTL2a@) does not change. (The library must never emit an @RTL2a@ state change event for a state equal to the previous state).
```

Comment:
As above, should we not be more explicit about the `ChannelState` and `ChannelEvent` types?